scripts/resource/Station.gd

```diff
@@ -74,19 +77,24 @@ func update_hydrogen() -> void:
 func update_max_hydrogen() -> void:
 	var total_storage_tiles = 0
 	for room in rooms:
-		if room.roomType.id == STORAGE_BAY_ID:
+		if room.roomType.id == ROOMTYPE.STORAGE_BAY:
 			total_storage_tiles += room.calculate_tile_count(room.topLeft, room.bottomRight)
 	max_hydrogen = 100 + (total_storage_tiles * 10)
+
+func add_crew(crew_member: CrewMember) -> void:
+	crew.append(crew_member)
+	crew_updated.emit(crew.size())
+
+func remove_crew(crew_member: CrewMember) -> void:
+	crew.erase(crew_member)
+	crew_updated.emit(crew.size())
 	
 func add_room(room: Room) -> void:
-	var new_rooms: Array[Room] = rooms.duplicate()
-	new_rooms.append(room)
-	rooms = new_rooms
+	rooms.append(room)
```

Comment:
Oh, this worked? You didn't need to recreate the whole array?

Reply:
yes, did not seem to have any issues 👍